src/cyminmax.pxd

```diff
@@ -2,6 +2,10 @@ cimport cython
 cimport numpy
 
 
+cdef extern from "numpy/npy_math.h":
+    bint npy_isnan(double x) nogil
```

Comment:
Would use Cython's copy, but it seems to want to use the GIL with it. So had to roll our own.

xref: https://github.com/cython/cython/issues/1989